src/server/constants/AutoComplete.ts

```diff
@@ -12,3 +12,22 @@ export const EDITABLE_ATTRIBUTES = ['class', 'default', 'escape', 'liquid', 'tag
 export const PORTAL_FILTERS = ['concat', 'except', 'first', 'join', 'last', 'order_by', 'random'];
 
 export const AUTO_COMPLETE_PLACEHOLDER = '_AUTO_COMPLETE_PLACEHOLDER_'
+
+export const OBJECT_ATTRIBUTES_MAP = new Map([
+    ['ads', ['placements']],
+    ['blogs', ['posts']],
+    ['entitylist', ['create_enabled', 'create_url', 'detail_enabled', 'detail_id_parameter', 'detail_label', 'detail_url', 'empty_list_text', 'enable_entity_permissions', 'entity_logical_name', 'filter_account_attribute_name', 'filter_apply_label', 'filter_definition', 'filter_enabled', 'filter_portal_user_attribute_name', 'filter_website_attribute_name', 'language_code', 'page_size', 'primary_key_name', 'search_enabled', 'search_placeholder', 'search_tooltip', 'views']],
+    ['entityview', ['columns', 'entity_permission_denied', 'entity_logical_name', 'first_page', 'Id', 'language_code', 'last_page', 'name', 'next_page', 'Page', 'pages', 'page_size', 'previous_page', 'primary_key_logical_name', 'records', 'sort_expression', 'total_pages', 'total_records']],
+    ['events', ['occurences']],
+    ['forloop', ['first', 'index', 'index0', 'Last', 'length', 'rindex', 'rindex0']],
+    ['forums', ['threads', 'All', 'thread_count', 'post_count']],
+    ['knowledge', ['articles', 'categories']],
+    ['language', ['url', 'url_substitution', 'name', 'code']],
```

Comment:
What's the need to keeping the attributes of each key?

Reply:
Here we are maintaining the attributes associated with each portal object. For example, when a user does {{ request. }} or {{ 
request[ }} & looks for suggestion(request being the object), we provide them based on this map.